carbon-design-system/carbon
# feat(colors): add hover colors to `@carbon/colors`

Closes #8822

This PR updates our `@carbon/colors` package to include the hover palette from the IDL. It also updates our build task to export hover variables in Sass.

#### Changelog

**New**

- Hover colors from the IDL

**Changed**

- Update build taskt to emit hover colors

**Removed**

#### Testing / Reviewing

- Pull down the PR locally
- Make sure to run `yarn build` in `@carbon/colors`
- Verify that the `scss/mixins.scss` file includes the hover colors from the IDL
- Verify that the `index.scss` file includes the hover colors from the IDL
